packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -19,6 +19,7 @@ import type {
     AppState,
 } from '@suite-types';
 import type { DebugModeOptions, AutodetectSettings } from '@suite-reducers/suiteReducer';
+import { TorStatus } from '@suite-reducers/suiteReducer';
```

Comment:
Nit: Move above type

Reply:
Thats why I said to move it above `import type` 😆

Reply:
0905e85e7